frontend/src/components/maps/MapFilterBar.tsx

```diff
@@ -76,6 +79,7 @@ export type MapFilterChangeEvent = {
   classificationId: string;
   minLotSize: string;
   maxLotSize: string;
+  inSurplusPropertyProgram: boolean;
```

Comment:
Wouldn't this be a nullable boolean?  We only want to include it if it's set to `true`.  Again, probably not required to be changed.

Reply:
Oh yes you're right